source/external-interface-design/external-interface-design.rst

```diff
@@ -126,22 +126,21 @@ in Chirpstack.
 Data from IoT devices is received by Chirpstack and sent to OS2iot by
 publishing to a MQTT broker which has OS2iot as a subscriber.
 
-Data sent to IoT devices is sent from OS2iot to Chirpstack using a
-the REST API. Communication between the IoT devices, gateways
+Data sent to IoT devices is sent from OS2iot to Chirpstack using gRPC. Communication between the IoT devices, gateways
 and Chirpstack is out of scope of this project.
 
 |image6|
 
-The Chirpstack is comprised of the following 4 modules.
+The Chirpstack is comprised of the following modules. In Chirpstack V4, the Application Server and Network Server is merged into one component.
```

Comment:
Could probably use some restructuring:
In Chirpstack V4, the Application and network server has been merged into the same component.